src/main/java/frc/team3388/robot/EncoderSrx.java

```diff
@@ -0,0 +1,29 @@
+package frc.team3388.robot;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import java.util.function.DoubleSupplier;
+
+public class EncoderSrx implements DoubleSupplier {
```

Comment:
Need to add a `getVelocity()` that will return the velocity in rpm for our PID implementation

Reply:
in prm